packages/astro/src/compiler/transform/styles.ts

```diff
@@ -107,7 +107,10 @@ async function transformStyle(code: string, { logging, type, filename, scopedCla
       css = code;
       break;
     }
-    case 'sass':
+    case 'sass': {
+      css = sass.renderSync({ data: code, includePaths, indentedSyntax: true }).css.toString('utf8');
```

Comment:
Thanks for adding this! But would you mind keeping the command combined, as it was before?

```js
      css = sass.renderSync({ data: code, includePaths, indentedSyntax: styleType === 'sass' }).css.toString('utf8');
```

Reason is that when separating these, there’s a good chance that someone would accidentally change behavior for one but not the other. But by keeping them combined, we guarantee both flavors will always get the same settings.

Reply:
Sounds good 👍

Reply:
Added in 7852c4a